### DISCUSSION TITLE
Error when querying the blockchain failed to load message: blockstore: block not found

### DISCUSSION THREAD

**Discussion by AgwaB** [filecoin-project/lotus#6744] [General Q&A]
The height of the latest tipset on the mainnet at the present time is 928360, and when the tipset is searched with a height of 900000, block `bafy2bzacebntc3dv3rxe5jyktqi2j2gyr4qjmgqmfrlpluqyznip7etfkpedi` is searched.

And when search this `bafy2bzacebntc3dv3rxe5jyktqi2j2gyr4qjmgqmfrlpluqyznip7etfkpedi` block with `ChainGetBlockMessages` rpc call, the following error appears.

`{"jsonrpc":"2.0","id":1,"error":{"code":1,"message":"failed to load msgmeta (bafy2bzacedijabl4g6mek3h2rxygzbcrdefdb3gsmucamrw47tk5tvukur7hu): blockstore: block not found"}}`

And when I try to find the receipt of the message included in this block, I get the following error similarly.
`{"jsonrpc":"2.0","id":1,"error":{"code":1,"message":"failed to load message: blockstore: block not found"}}`

I think I'm missing something about the filecoin concept, can you help?

**Comment by arajasek** [comment]
@AgwaB How did you sync your Lotus node? Was it from a snapshot? If so, you will not have "state" for tipsets before the snapshot point.

**Comment by LexLuthr** [comment]
@AgwaB Do you still need help with this?
